packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -215,7 +215,12 @@ void main() {
         child: const Text('XXX'),
       ),
     );
-    final WidgetPredicate popupMenu = (Widget widget) => widget.runtimeType.toString() == '_PopupMenu';
+    final WidgetPredicate popupMenu = (Widget widget) {
+      final String widgetType = widget.runtimeType.toString();
+      // Dart 2 implements reified generic methods which means showMenu<int>(...)
+      // creates _PopupMenu<int> and not _PopupMenu<dynamic>.
```

Comment:
This might be clearer for people coming to the code without context:

```dart
// TODO(mraleph): Remove the old case below.
return widgetType == '_PopupMenu<int>' // normal case
    || widgetType == '_PopupMenu'; // for old versions of Dart that don't reify method type arguments
```